service/src/main/java/org/cbioportal/genome_nexus/util/GenomicVariant.java

```diff
@@ -0,0 +1,30 @@
+package org.cbioportal.genome_nexus.util;
+public class GenomicVariant {
+    public GenomicVariantUtil util = new GenomicVariantUtil();
```

Comment:
would be good to read up on the concept of util classes: https://stackoverflow.com/questions/25223553/how-can-i-create-an-utility-class. Also there's some people that advocate against using util classes with static methods: https://www.vojtechruzicka.com/avoid-utility-classes/. They propose to just instantiate it once, which could be a better solution